$NetBSD: patch-jdk_make_lib_NioLibraries.gmk,v 1.4 2015/09/24 22:54:28 tnn Exp $

Fix list of platforms that do not have SCTP support.

--- jdk/make/lib/NioLibraries.gmk.orig	2015-06-10 10:36:24.000000000 +0000
+++ jdk/make/lib/NioLibraries.gmk
@@ -185,10 +185,12 @@ $(BUILD_LIBNIO): $(BUILD_LIBNET)
 
 ##########################################################################################
 
+ifeq (, $(OS_VENDOR))
+OS_VENDOR:=    $(shell uname -s)
+endif
 ifeq ($(OPENJDK_TARGET_OS_API), posix)
 
-  ifeq (, $(filter $(OPENJDK_TARGET_OS), macosx aix))
-  ifeq (, $(filter $(OPENJDK_TARGET_OS_VENDOR), netbsd openbsd))
+  ifeq (, $(filter $(OS_VENDOR), AIX Darwin NetBSD OpenBSD DragonFly))
 
     # Suppress unused parameters required by exported JNI functions.
     SCTP_WERROR := -Werror -Wno-error=unused-parameter
@@ -224,5 +226,4 @@ ifeq ($(OPENJDK_TARGET_OS_API), posix)
 
     $(BUILD_LIBSCTP): $(BUILD_LIBNIO)
   endif
-  endif
 endif
